cves/kernel/CVE-2016-3135.yml

```diff
@@ -452,7 +469,13 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    The mistake made is one that all coders run into at some point. Failing
+    to recognize situations where inputs could break the system is easy to
+    come across and often takes a lot of extra time and brainpower to find.
+    This vulnerability seems to have been brought about by inattention. The
+    CWE-190 suggests mitigations like strictly defining the bounds and this was
+    not performed correctly until the fix.
```

Comment:
Great answer! However I think it would be helpful to categorize and explicitly mention if it was a slip, lapse or planning error.

Reply:
Now mentions that is was a lapse.